build-core-armv7a7-imx6ull.sh

```diff
@@ -45,9 +45,10 @@ make -C "phoenix-rtos-posixsrv" all install
 
 # FIXME: compile host tools using host-pc target?
 b_log "Building hostutils"
-make -C "phoenix-rtos-hostutils" -f Makefile.old $CLEAN all
+make -C "phoenix-rtos-hostutils" -f Makefile.old "$CLEAN" all
```

Comment:
no, don't quote clean! (make will fail without "clean" then).

Do we need to pass clean to the make here? I assume the problem is the generic rm in build.sh does not touch hostutils build dir?

Reply:
Nah, it removes the hostutils build dir so passing `clean` here should not be needed:
![image](https://user-images.githubusercontent.com/3939983/151977152-0390ed04-2a4e-4f28-8900-ca312279e494.png)